docs/api-guide/caching.md

```diff
@@ -28,34 +28,31 @@ from rest_framework import viewsets
 
 class UserViewSet(viewsets.ViewSet):
     # With cookie: cache requested url for each user for 2 hours
-    @method_decorator(cache_page(60*60*2))
+    @method_decorator(cache_page(60 * 60 * 2))
     @method_decorator(vary_on_cookie)
     def list(self, request, format=None):
-        content = {
-            'user_feed': request.user.get_user_feed()
-        }
```

Comment:
maybe we can do
```python
 content = {
            'user_feed': request.user.get_user_feed(),
     }
```

Reply:
This is something changed by the command, not a manual change.

Reply:
Please add a comma at  the end to make it look identical to old version